proxy/src/main/java/org/apache/rocketmq/proxy/metrics/ProxyMetricsManager.java

```diff
@@ -69,6 +69,9 @@ public class ProxyMetricsManager implements StartAndShutdown {
     public static ObservableLongGauge proxyUp = null;
 
     public static void initLocalMode(BrokerMetricsManager brokerMetricsManager, ProxyConfig proxyConfig) {
+        if (proxyConfig.getMetricsExporterType() != BrokerConfig.MetricsExporterType.DISABLE) {
```

Comment:
We should skip init process when metrics exporter is disable and not the other way around.

Reply:
This check should also be added in ProxyMetricsManager#start.

Reply:
> This check should also be added in ProxyMetricsManager#start.
ProxyMetricsManager#start has existed

Reply:
I will fix some problems and submit again.

Reply:
@lizhimins thanks for you modify this error